index.html

```diff
@@ -327,7 +350,7 @@
                 </tr>
                 <tr>
                   <td class="tblhead">Release Date</td>
-                  <td>Expecting to release before 2022-MAR-25</td>
+                  <td>Expecting to release before 2022-APR-20</td>
```

Comment:
How about wording it as: 

"Expected to be released before 2022-APR-20"

Reply:
Like that, thank you Jasper :)